### DISCUSSION TITLE
Problem with Stitching sample,  stitch() status code = 1

### DISCUSSION THREAD

**Discussion by JeeDevUser** [bytedeco/javacv#1703] [Q&A] [answered]
Hi,
I am just trying to run sample from
    https://github.com/bytedeco/sample-projects/tree/master/opencv-stitching

In the class
  https://github.com/bytedeco/sample-projects/blob/master/opencv-stitching/src/main/java/Stitching.java  
following line:
 `int status = stitcher.stitch(imgs, pano);   `
 
 always return status code  = 1
 
 I have no idea whats going wrong, nor how to determine exactly what the problem is .
 Any suggestion/help?

**Comment by JeeDevUser** [comment, marked as answer]
Sorry, my mistake...

**Reply by saudet** [reply]
So, what was the issue?

**Comment by JeeDevUser** [comment]
My mistake, sorry

pet, 1. okt 2021. 15:36 Samuel Audet ***@***.***> је
написао/ла:
